gender.go

```diff
@@ -14,9 +14,9 @@ func (g Gender) String() string {
 	if g == Genderless {
 		return ""
 	} else if g == Female {
-		return "\u2640"
+		return "♀"
```

Comment:
how did you do that?

Reply:
Copy paste the character from the internet 😮

Reply:
oh lmao